File: algorithms/searching/linear-search/linear-search.go
package main

import (
	"fmt"
)

func main() {
	arr := []int{1, 2, 3, 4, 5, 6, 67, 8}
	fmt.Println(linearSearch(arr, 1))

}

func linearSearch(arr []int, value int) bool {
	for i := 0; i < len(arr); i++ {
		if arr[i] == value {
			return i
		}
	}
	return -1
}

File: algorithms/searching/linear-search/linear-search.js
var arr = [1, 2, 3, 4, 5, 6, 67, 8];

function linearSearch(a, value) {
    for (let i = 0; i < a.length; i++) {
        if (a[i] === value) {
            return i;
        }
    }

    return -1;
}

console.log(linearSearch(arr, 1));
